tt_metal/include/compute_kernel_api/eltwise_unary/eltwise_unary.h

```diff
@@ -31,8 +31,8 @@ ALWI void unary_op_init_common(uint32_t icb, uint32_t ocb = 16)
     MATH(( llk_math_hw_configure_disaggregated(icb, icb) ));
 }
 
-ALWI void init_sfpu(uint32_t icb) {
-    unary_op_init_common(icb);
+ALWI void init_sfpu(uint32_t icb, uint32_t ocb = 16) {
```

Comment:
is there anywhere I can read about why output operands start at index 16?

Reply:
I don't know the origin here or why this was done.   I can see in the code that we had implied types based on the enum value.   I speculated that it was to simply give everyone commonality so that different kernel dependencies would use the same set of indexes between host and kernel side code.  @rdjogoTT do you know why this was the case years ago?

Reply:
Input operands were 0-7, param operands were 8-15, outputs 16-23, and intermediates 24-31. The distinction between the 4 types used to be necessary, I think because of the way streams used to be utilized in old stacks. You can try asking someone who has been here longer for details.